java/pom.xml

```diff
@@ -395,6 +395,14 @@
                                 fail("Could not find cudf as a dependency of libcudfjni out> $sout err> $serr")
                             }
 
+                            def libnvcomp = ~/libnvcomp\\.so\\s+=>\\s+(.*)libnvcomp.*\\.so\\s+.*/
+                            def nvcompm = libnvcomp.matcher(sout)
+                            if (nvcompm.find()) {
+                                pom.properties['native.nvcomp.path'] = nvcompm.group(1)
+                            } else {
+                                fail("Could not find nvcomp as a dependency of libcudfjni out> $sout err> $serr")
```

Comment:
The existing message is more accurate.  The code is listing the dependencies of libcudfjni, not libnvcomp, looking for the various libraries.  If it can't find the nvcomp library (i.e.: libnvcomp) then it means it wasn't a dependency of libcudfjni as the existing message indicates.